.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Vets-Who-Code",
+  "image": "vets-who-code:latest",
+  "workspaceFolder": "/src",
+  "mounts": [
+    "source=${localWorkspaceFolder},target=/src,type=bind,consistency=cached",
+    "source=node_modules,target=/src/node_modules,type=volume"
+  ],
+  "customizations": {
+    "vscode": {
+      "settings": {
+        "terminal.integrated.shell.linux": "/bin/bash"
+      },
+      "extensions": [
+        "dbaeumer.vscode-eslint",
+        "ms-vscode.vscode-typescript-tslint-plugin"
+      ]
+    }
+  },
+  "forwardPorts": [3000],
```

Comment:
Should this be a more unique port to avoid collisions with other processes?

Reply:
I'm not super familiar with dev containers to know if this is self contained inside the container or would collide with other  processes outside it.

Reply:
This exposes running process on 3000, it's standard to directly expose it like this.

Reply:
@tupleHunden Port 3000 is standard for this service and is isolated within the dev container, so it shouldn't collide with external processes.